administrator/components/com_content/tmpl/articles/blankstate.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_content
+ *
+ * @copyright   (C) 2008 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Router\Route;
+
+?>
+<form action="<?php echo Route::_('index.php?option=com_content&view=articles'); ?>" method="post" name="adminForm" id="adminForm">
+
+	<div class="px-4 py-5 my-5 text-center">
+		<span aria-hidden="true" class="fa-8x icon-copy mb-4 article"></span>
```

Comment:
```suggestion
		<span class="fa-8x icon-copy mb-4 article" aria-hidden="true" ></span>
```

Just for consistency